DependencyInjection/SonataAdminExtension.php

```diff
@@ -90,6 +90,7 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setParameter('sonata.admin.configuration.dashboard_blocks', $config['dashboard']['blocks']);
         $container->setParameter('sonata.admin.configuration.sort_admins', $config['options']['sort_admins']);
         $container->setParameter('sonata.admin.configuration.breadcrumbs', $config['breadcrumbs']);
+        $container->setParameter('sonata.admin.configuration.role_sonata_admin', $config['security']['roles']['role_sonata_admin']);
```

Comment:
But I'm not 100% sure it's the right thing to do (might look more complicated because it does not use the service definition file). WDYT @sonata-project/contributors ?